bokehjs/src/lib/models/canvas/canvas.ts

```diff
@@ -181,7 +185,12 @@ export class CanvasView extends DOMView {
     extend(this.el.style, style)
     append(this.el, ...elements)
 
-    logger.debug("CanvasView initialized")
```

Comment:
This could be a trace

Reply:
Most (if not all) of those debug/trace log statements are a waste of bundle space. Recently all major browsers added "log points" to their debuggers, so one can trace code execution without changing the code or inflating bundle size.